src/Components/Facility/AssetCreate.tsx

```diff
@@ -66,51 +63,75 @@ const goBack = () => {
 
 const AssetCreate = (props: AssetProps) => {
   const [state, dispatch] = useReducer(asset_create_reducer, initialState);
+  const [name, setName] = useState<string>("");
+  const [asset_type, setAssetType] = useState<string>("");
+  const [description, setDescription] = useState<string>("");
+  const [is_working, setIsWorking] = useState<string>();
+  const [serial_number, setSerialNumber] = useState<string>("");
+  const [warranty_details, setWarrantyDetails] = useState<string>("");
+  const [location, setLocation] = useState<string>("");
```

Comment:
btw is it good idea to useState for each field separately ?
I think using `state.form` is standard as in [doc](https://reactjs.org/docs/forms.html#handling-multiple-inputs)
If this is done because of repeated typing of `state.form` then we can first assign these to short name local variables then use them in other places.

Reply:
No I was having problem with using it for both update and create, so I had to change from state.form to this...